main/lib/src/DataCollector.cc

```diff
@@ -183,7 +183,8 @@ namespace eudaq {
         if ((m_buffer[i].events.front()->GetEventNumber() != m_eventnumber) &&
             (m_buffer[i].events.front()->GetEventNumber() !=
              m_eventnumber - 1)) {
-          if (ev.GetEventNumber() % 1000 == 0) {
+	  //tests 12/05  
+	  if (ev.GetEventNumber() % 1000 == 0) {
```

Comment:
True: is not changed, we did some tests and commited to the repository by mistake